src/formatDate.js

```diff
@@ -50,7 +50,31 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateStr = date.split(fromFormat[3]);
```

Comment:
Magic number 3?

Reply:
Date always have the same length, so there's no need to create a variable or .length method)